router/batchrouter/asyncdestinationmanager/sftp/manager.go

```diff
@@ -0,0 +1,97 @@
+package sftp
+
+import (
+	"encoding/json"
+	"path/filepath"
+
+	"github.com/rudderlabs/rudder-go-kit/logger"
+	sftp "github.com/rudderlabs/rudder-go-kit/sftp"
+	backendconfig "github.com/rudderlabs/rudder-server/backend-config"
+	"github.com/rudderlabs/rudder-server/jobsdb"
+	"github.com/rudderlabs/rudder-server/router/batchrouter/asyncdestinationmanager/common"
+	"github.com/tidwall/gjson"
+)
+
+func (d *DefaultManager) Transform(job *jobsdb.JobT) (string, error) {
+	return common.GetMarshalledData(job, func(payload json.RawMessage) string {
+		return string(payload)
+	}), nil
+}
+
+// Upload uploads the data to the destination and marks all jobs to be completed
+func (d *DefaultManager) Upload(asyncDestStruct *common.AsyncDestinationStruct) common.AsyncUploadOutput {
+	destination := asyncDestStruct.Destination
+	filePath := asyncDestStruct.FileName
+	destinationID := destination.ID
+	destConfigJSON, err := json.Marshal(destination.Config)
+	if err != nil {
+		return generateErrorOutput("Error while marshalling destination config ", err, asyncDestStruct.ImportingJobIDs, destinationID)
+	}
+
+	uploadFilePath := gjson.Get(string(destConfigJSON), "filePath").String()
```

Comment:
uploadFilePath -> uploadFilePathTemplate
"filePath" -> should be make "filePathPrefix"

Reply:
`filePath` is the path in the input in UI from user.
<img width="456" alt="Screenshot 2024-04-25 at 11 13 44 AM" src="https://github.com/rudderlabs/rudder-server/assets/60897972/0ed93c9b-35a3-48f9-b297-dba8963930f4">